sdk/src/client/node_api/mqtt/mod.rs

```diff
@@ -206,11 +206,18 @@ fn poll_mqtt(client: &Client, mut event_loop: EventLoop) {
                                         let payload = &*p.payload;
                                         let protocol_parameters = &client.network_info.read().await.protocol_parameters;
 
-                                        match MilestonePayload::unpack_verified(payload, protocol_parameters) {
-                                            Ok(milestone_payload) => Ok(TopicEvent {
+                                        match Payload::unpack_verified(payload, protocol_parameters) {
+                                            Ok(Payload::Milestone(milestone)) => Ok(TopicEvent {
                                                 topic: p.topic.clone(),
-                                                payload: MqttPayload::MilestonePayload(milestone_payload),
+                                                payload: MqttPayload::MilestonePayload(*milestone),
                                             }),
+                                            Ok(p) => {
+                                                warn!(
+                                                    "'milestone' topic returned non-milestone payload, kind: {:?}",
+                                                    p.kind()
+                                                );
+                                                Err(())
+                                            }
```

Comment:
Better just remove this?

Reply:
Its needed for the match now, or you just want to ignore it?

Reply:
Ah ok because with _ we cant get the error message, it's fine this way then